TODO.md

```diff
@@ -15,17 +15,22 @@
 
 ## Voice / WebRTC
 
-- [ ] Add a microphone boost slider.
-- [ ] Add a speaker adjustment slider.
-- [ ] Add individual adjustment sliders to each of the players.
+- [x] Add a microphone boost slider.
+- [x] Add a speaker adjustment slider.
+- [x] Add individual adjustment sliders to each of the players.
+- [x] Add a OBS Overlay
+- [x] Add a Android support
+- [ ] Add a iOS Support 
+- [x] Add a option in "Host Settings" Can hear through cams.
```

Comment:
Add **an** option in "Host Settings" **for hearing** through cams

Reply:
Resolved thanks :)